x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/service/ElasticServiceAccounts.java

```diff
@@ -157,7 +157,7 @@ final class ElasticServiceAccounts {
             new String[] { "monitor", "manage_own_api_key" },
             new RoleDescriptor.IndicesPrivileges[] {
                 RoleDescriptor.IndicesPrivileges.builder()
-                    .indices("logs-*", "metrics-*")
+                    .indices("logs-*", "metrics-*", "traces-*")
```

Comment:
From the description it looks like we would also need to add something for `.logs-endpoint*`?

Reply:
Oh, I see it was added but reverted as not needed https://github.com/elastic/elasticsearch/pull/103445/commits/c88a773891106080decbafef01550a34d00a1afd

Reply:
yes, updated the description